### PR TITLE
Add documentation about release creation.

### PR DESCRIPTION
**Pull request by ouphi**
Document how to create a release by using the Prerelease workflow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,3 +117,15 @@ Refer to the [Dev Guide][devguide] for development and debugging instructions.
 [knownissues]:          https://azure.github.io/iotedge-lorawan-starterkit/dev/issues/
 [partnerinstructions]:  https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/partner/
 [specificationsupport]: https://azure.github.io/iotedge-lorawan-starterkit/dev/#lorawan-specification-support
+
+## Create a release
+
+You can create a prerelease with the following steps:
+
+- Run the Prerelease workflow
+  Go to `Actions`, select `Prerelease` action, then select `Run workflow` and specify the release version before running the workflow.
```
Comment (Mandur) [inline README.md:126]: nit. we could put a direct link to the workflow

File: README.md [hunk 2]
```diff
@@ -117,3 +117,15 @@ Refer to the [Dev Guide][devguide] for development and debugging instructions.
...
+- Merge 2 PRs
+  The Prerelease workflow will create 2 PRs, one PR to update the Starter Kit version in Bicep and one PR to update the Button URL. Merge these 2 PRs.
+- Add a release description and set as the latest
```
Comment (Mandur) [inline README.md:129]: I am not sure I understand here, Do you want to say "create a git tag pointing to the latest commit on the master branch"?

Comment (Mandur) [reply README.md:129]: In this case there is a step earlier, which is pushing dev to master

File: README.md [hunk 3]
```diff
@@ -117,3 +117,15 @@ Refer to the [Dev Guide][devguide] for development and debugging instructions.
...
 [partnerinstructions]:  https://azure.github.io/iotedge-lorawan-starterkit/dev/user-guide/partner/
 [specificationsupport]: https://azure.github.io/iotedge-lorawan-starterkit/dev/#lorawan-specification-support
+
+## Create a release
+
+You can create a prerelease with the following steps:
+
+- Run the Prerelease workflow
+  Go to `Actions`, select `Prerelease` action, then select `Run workflow` and specify the release version before running the workflow.
+- Merge 2 PRs
```
Comment (Mandur) [inline README.md:127]: Looks duplicate with the content you have in the line below?

**Review by Mandur — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (ouphi) [reply README.md]: These are supposed to be small titles, I changed them to be markdown heading instead to be clearer

**Review by Mandur — APPROVED** [review]

**Review by Mandur — APPROVED** [review]

**Pull request merged**